glide.yaml

```diff
@@ -9,16 +9,16 @@ import:
   version: 773f5027234d0b08adf766be34f55df2f312abf7
 - package: github.com/kelseyhightower/confd
   repo: https://github.com/projectcalico/confd.git
-  version: 7201c095f0b27b1e1173a254244f5aee9cf3dd46
+  version: a5c3629c9c3545870f60ab56e31d411849518ce1
```

Comment:
I don't think this is correct. The latest in the release-v3.8 branch of confd is the previous version